### PR TITLE
Use new tooling system

### PR DESCRIPTION
**Pull request by woksin**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1115616103797671/1128288502381487)

### CODE REVIEW

**Review by einari — CHANGES_REQUESTED** [review]
Looks good. 
I've put "Request changes" - more so that we can have a conversation.
Curious if we can find a better naming strategy. 2 things that jump out at me; prefixes and postfixes. 
There are a few things being prefixed with **Cli** e.g. `CliCommand`, `CliCommandManager` and more. I see that it does in fact inherit a base `Command` for the `CliCommand` for instance. But it feels a bit weird to prefix things with **Cli** when we're in the CLI project. The second is the postfix; e.g. Manager (https://dolittle.io/contributing/guidelines/naming/). Could it be just `Commands`.

Other than that, this is looking real good other than the fact that it is a massive PR; we've talked about this - but just want to remind us that this is not the normal and moving forward PRs should be kept to a feature level. This is basically a big bang rewrite and introduction of a bunch of features.

**Review by einari — CHANGES_REQUESTED** [review]
IMO; I think this looked even worse. 
There is something not feeling quite right here. `Wrapped*` prefix is telling us that we are wrapping something. While in the context of the CLI we don't really care. So ideally it is basically a just for instance a `Command` - but we're importing it from the CLI version. Then the CLI version needs to be implementing the correct base type / interface.

Suggestion (If Typescript permits - haven't checked)

```typescript
import Command as BaseCommand from '@dolittle/....';
```

or 

```typescript
import * as BaseCommand from '@dolittle/....';
```

Bottom line is that we're not building WrappedCommand, WrappedCommandGroup or WrappedNamespace - we're building the CLI specific Command, CommandGroup and Namespace

File: Source/Commands/Commands.ts [hunk 1]
```diff
@@ -11,51 +11,45 @@ import chalk from 'chalk';
...
     
     constructor(private _commandManager: ICommandManager, private _dependencyResolvers: IDependencyResolvers) {
         this._commands = [
-            new Init(),
+            // new Init(),
             new Check()
         ];
 
-        this.createCliCommands();
+        this.createWrappedCommands();
```
Comment (einari) [inline Source/Commands/Commands.ts]: Sould just be `createCommands`

**Comment by woksin** [comment]
Yeah, see your point lol.

**Comment by woksin** [comment]
That should do it, @einari

**Review by einari — APPROVED** [review]
Looking great!
